pkg/verify/verify.go

```diff
@@ -151,16 +72,28 @@ type ClientBuilder interface {
 	HTTPClient() (*http.Client, error)
 }
 
-// simpleClientBuilder implements the ClientBuilder interface and should be used for testing.
+// DefaultClient uses the default http.Client for accessing signatures.
+var DefaultClient = simpleClientBuilder{}
+
+// simpleClientBuilder implements the ClientBuilder interface and may be used for testing.
 type simpleClientBuilder struct{}
 
 // HTTPClient from simpleClientBuilder creates an httpClient with no configuration.
-func (s *simpleClientBuilder) HTTPClient() (*http.Client, error) {
+func (s simpleClientBuilder) HTTPClient() (*http.Client, error) {
```

Comment:
Why are we dropping the pointer collector?

Reply:
You don't need pointer on static types (which this is)